rb/lib/selenium/webdriver/common/port_prober.rb

```diff
@@ -38,7 +38,7 @@ def self.random
         port
       end
 
-      IGNORED_ERRORS = [Errno::EADDRNOTAVAIL]
+      IGNORED_ERRORS = [Errno::EADDRNOTAVAIL].freeze
```

Comment:
We can't freeze this and then add something to it in the next line. The `freeze` on line 43 is sufficient. If Rubocop flags this we need to exclude it.

Reply:
@titusfortner

I think so, too.
Added setting to exclude it in rubocop.yml.